app.js

```diff
@@ -0,0 +1,43 @@
+    { q: "Кругосветное путешествие не совершал:", ans1: "1) Фернан Магеллан", ans2: "2) Фрэнсис Дрейк", ans3: "3) Джеймс Кук", ans4: "4) Юрий Гагарин", corAns: 1, sumCond: 2000, sumUncond: 0 },
+    { q: "'Камикадзе' в переводе с японского языка означает:", ans1: "1) Крутое пике", ans2: "2) Божественный ветер", ans3: "3) Планы на вечер", ans4: "4) Верный долгу", corAns: 2, sumCond: 3000, sumUncond: 0 },
+    { q: "Гарантом брачного союза в Древнем Риме был(а):", ans1: "1) Юпитер", ans2: "2) Венера", ans3: "3) Зевс", ans4: "4) ЗАГС", corAns: 1, sumCond: 5000, sumUncond: 5000 },
+    { q: "'Галактика' в переводе с древне-греческого языка переводится как:", ans1: "1) Вечный Зов", ans2: "2) Млечный Путь", ans3: "3) Вечерний Звон", ans4: "4) Ложись Спать", corAns: 2, sumCond: 8000, sumUncond: 5000 },
+    { q: "Свежевыжатый неосветлённый сок:", ans1: "1) Осло", ans2: "2) Сусло", ans3: "3) Щячло", ans4: "4) Поньо", corAns: 2, sumCond: 10000, sumUncond: 5000 },
+    { q: "На голову надевают:", ans1: "1) Гитару", ans2: "2) Тиару", ans3: "3) Гагару", ans4: "4) Гетеру", corAns: 2, sumCond: 13000, sumUncond: 5000 },
+    { q: "'Їхав козак за Дунай, сказав дівчині ...'", ans1: "1) Давай", ans2: "2) Стріляй", ans3: "3) Прощй", ans4: "4) Тікай", corAns: 3, sumCond: 15000, sumUncond: 5000 },
+    { q: "Самая большая рыба на планете", ans1: "1) Касатка", ans2: "2) Китовая акула", ans3: "3) Синий кит", ans4: "4) Кашалот", corAns: 2, sumCond: 25000, sumUncond: 25000 },
+    { q: "Продолжите цитату Э.Хемингуэя - 'Чтобы окружающие меня люди становились интереснее, я ...'", ans1: "1) молчу", ans2: "2) пишу", ans3: "3) пою", ans4: "4) пью", corAns: 4, sumCond: 50000, sumUncond: 25000 },
+    { q: "Столица Мадагаскара", ans1: "1) Антананариву", ans2: "2) Аста ла Виста", ans3: "3) Каракас", ans4: "4) Мучачос", corAns: 1, sumCond: 100000, sumUncond: 25000 },
+    { q: "Стеклянный графин, предназначенный для сливания и подачи вина:", ans1: "1) Рейхсвер", ans2: "2) Декантер", ans3: "3) Фарватер", ans4: "4) Газгольдер", corAns: 2, sumCond: 250000, sumUncond: 25000 },
+    { q: "Что из перечисленного является наукой:", ans1: "1) Астрология", ans2: "2) Алхимия", ans3: "3) Гомеопатия", ans4: "4) Обработка древесины", corAns: 4, sumCond: 500000, sumUncond: 25000 },
+    { q: "Наклон оси вращения Земли обуславливает:", ans1: "1) Смену времен года", ans2: "2) Наличие високосного года", ans3: "3) Смену дня и ночи", ans4: "4) Наличие полярного сияния", corAns: 1, sumCond: 1000000, sumUncond: 25000 },
+]
+
+for (let i = 0; i < arr.length; ++i) {
+    let corAnsArr = [1, 2, 3, 4];
+    let a;
+    do {
+        a = +prompt(`${arr[i].q} \n ${arr[i].ans1} \n ${arr[i].ans2} \n ${arr[i].ans3} \n ${arr[i].ans4}`, 'Введите цифру, соответствующую ответу');
+        if (corAnsArr.includes(a) == false)
+            alert(`Повторите ввод (1, 2, 3 или 4)`);
+    }
+    while (corAnsArr.includes(a) == false);
```

Comment:
please, do not use '==' and '!=' operators!

`corAnsArr.includes(a) == false` is the same as `!corAnsArr.includes(a)`